docs/root/start/sandboxes/templating.rst

```diff
@@ -0,0 +1,112 @@
+.. _install_sandboxes_templating:
+
+Templating
+==========
+
+In this example, we show how to template Envoy's YAML file using environment variables.
```

Comment:
This is for static config only? Can we be explicit here?

Reply:
I think we can leverage this example to do a dynamic file reloading, e.g. using path for eds https://www.envoyproxy.io/docs/envoy/latest/api-v2/api/v2/core/config_source.proto#core-configsource

Reply:
@dio Yes, at the moment, the example was for static configs.  I can definitely be more explicit in the docs.  But, I can also explore how to integrate dynamic file reloading into this example.